src/App/Program.cs

```diff
@@ -54,10 +54,19 @@ void ConfigureServices(IServiceCollection services, IConfiguration config)
 
     // HttpClients for platform functionality. Registered as HttpClients so default HttpClientFactory is used
     services.AddHttpClient<AuthorizationApiClient>();
-    services.AddAltinnAppServices(config, builder.Environment);
-
-    // Altinn App implementation service (The concrete implementation of logic from Application repository)
-    services.AddTransient<IAppModel, Altinn.App.AppLogic.App>();
+    
+    // ##############################################################################################################
+    // # Add your services below here                                                                               #
+    // ##############################################################################################################
+    
+    // ##############################################################################################################
+    // # End of your services                                                                                       #
+    // ##############################################################################################################
+    services.AddAppServices(config, builder.Environment);
+    services.AddPlatformServices(config, builder.Environment);
```

Comment:
These two lines should be replaced with `services.AddAltinnAppServices(config, builder.Environment);`